spotinst_sdk2/test/test_lib/input/managed_instance/aws_managed_instance.json

```diff
@@ -111,12 +111,7 @@
         "loadBalancers": [
           {
             "name": "name",
-            "arn": "arn",
-            "type": "MULTAI_TARGET_SET",
-            "balancerId": "lb-1ee2e3q",
-            "targetSetId": "ts-3eq",
-            "azAwareness": true,
-            "autoWeight": true
+            "arn": "arn"
```

Comment:
type is required here, either "CLASSIC" or "TARGET_GROUP"

Reply:
Fixed.